llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -94,7 +99,8 @@ async def completion(
         response_format: Optional[ResponseFormat] = None,
         stream: Optional[bool] = False,
         logprobs: Optional[LogProbConfig] = None,
-    ) -> AsyncGenerator:
+    ) -> CompletionResponse | AsyncGenerator[CompletionResponseStreamChunk, None]:
+        assert self.model_store is not None
```

Comment:
using `assert` in non-test code? raise intead?

Reply:
It’s a convincing argument, but you already know that assert statements can be stripped out when Python runs with the optimized flag. Given that both raise and assert will exit the program, why not use raise instead? They achieve the same outcome, except assert is weaker since it can be ignored in production.

I guess I’m considering two scenarios:
* If we want to allow self.model_store = None and do nothing (as it behaves today), then using cast could be the right choice if we want to make mypy happy.
* If we believe None is an actual error, we should probably use raise instead of assert.

Does that make sense?